### PR TITLE
fix: Fixed markdown message in channel preview

### PR DESCRIPTION
**Pull request by liamcho**
Fixes [CLNP-4063](https://sendbird.atlassian.net/browse/CLNP-4063)

### Changelogs
- Fixed a bug where markdown messages are incorrectly displayed in channel preview

Screenshot 2024-08-05 at 12 34 49 PM

### Disabled
Screenshot 2024-08-01 at 3 16 11 PM
Screenshot 2024-08-01 at 3 19 25 PM

### Enabled
Screenshot 2024-08-01 at 3 10 04 PM
Screenshot 2024-08-01 at 3 19 33 PM

[CLNP-4063]: https://sendbird.atlassian.net/browse/CLNP-4063?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/modules/Message/utils/tokens/tokenize.ts
Comment (AhyoungRyu) [inline src/modules/Message/utils/tokens/tokenize.ts:1]: Could you add unit tests for the newly added ones?

Comment (liamcho) [reply src/modules/Message/utils/tokens/tokenize.ts:1]: ok

Comment (liamcho) [reply src/modules/Message/utils/tokens/tokenize.ts:1]: Added unit tests @AhyoungRyu

**Review by HoonBaek — APPROVED** [review]
LGTM

File: src/modules/GroupChannelList/components/GroupChannelListItem/utils.ts [hunk 1]
```diff
@@ -79,6 +79,8 @@ const getPrettyLastMessage = (message = null, stringSet = LabelStringSet) => {
   return message.message ?? '';
 };
 
-export const getLastMessage = (channel?: GroupChannel, stringSet = LabelStringSet) => channel?.lastMessage ? getPrettyLastMessage(channel?.lastMessage, stringSet) : '';
+export const getLastMessage = (channel?: GroupChannel, stringSet = LabelStringSet) => {
```
[the remark this answers is not in the document]
Comment (liamcho) [reply src/modules/GroupChannelList/components/GroupChannelListItem/utils.ts:82]: const { stringSet } = useLocalization();
을 함수 안에 추가하는 방향일까요?

[the remark this answers is not in the document]
Comment (liamcho) [reply src/modules/GroupChannelList/components/GroupChannelListItem/utils.ts:82]: getLastMessage 는 현재 쓰이고 있는 곳들이 많아서 나중에 따로 보면 좋을것 같습니다

**Pull request merged**